components/camel-azure/camel-azure-storage-blob/src/main/java/org/apache/camel/component/azure/storage/blob/CredentialType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.azure.storage.blob;
+
+public enum CredentialType {
+    /**
+     * Shared key credential
+     * @see com.azure.storage.common.StorageSharedKeyCredential
+     */
+    shared_key_credential,
```

Comment:
I think we should follow the constants convention and naming in uppercase

Reply:
I will fix the new one I created; However note that other enums in the components created outside of this PR don't follow the convention.
It may be worth opening a separate ticket to tidy-up the component to make it consistent.

Reply:
Fixed

Reply:
Please open an issue for this.